scripts/hybrid-custody/get_accessible_proxy_account_nfts.cdc

```diff
@@ -0,0 +1,78 @@
+import "HybridCustody"
+import "NonFungibleToken"
+import "MetadataViews"
+
+
+// This script iterates through a parent's child accounts, 
+// identifies private paths with an accessible NonFungibleToken.Provider, and returns the corresponding typeIds
+pub fun main(addr: Address):AnyStruct {
+  let account = getAuthAccount(addr)
+  let manager = getAuthAccount(addr).borrow<&HybridCustody.Manager>(from: HybridCustody.ManagerStoragePath) ?? panic ("manager does not exist")
+
+  var typeIdsWithProvider = {} as {Address: [String]} 
+  var nftViews = {} as {Address: {UInt64: MetadataViews.Display}} 
+
+  
+  let providerType = Type<Capability<&{NonFungibleToken.Provider}>>()
+  let collectionType: Type = Type<@{NonFungibleToken.CollectionPublic}>()
+
+  // Iterate through child accounts
+  for address in manager.getChildAddresses() {
+    let acct = getAuthAccount(address)
+    let foundTypes: [String] = []
+    let views: {UInt64: MetadataViews.Display} = {}
+    let childAcct = manager.borrowAccount(addr: address) ?? panic("child account not found")
+    // get all private paths
+    acct.forEachPrivate(fun (path: PrivatePath, type: Type): Bool {
+      // Check which private paths have NFT Provider AND can be borrowed
+      if !type.isSubtype(of: providerType){
+        return true
+      }
+      if let cap = childAcct.getCapability(path: path, type: Type<&{NonFungibleToken.Provider}>()) {
+        let providerCap = cap as! Capability<&{NonFungibleToken.Provider}> 
+
+        if !providerCap.check(){
+          return true
```

Comment:
same here for a comment, it could get confusing that `return true` is here for those who aren't familiar with storage iteration